### PR TITLE
Proxy VM Image Docs

### PR DESCRIPTION
**Pull request by jcayouette**
Adds Proxy VM deployment to documentation for 5.0.

### CODE REVIEW

**Review by 0rnela — APPROVED** [review]

File: modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vm-suma.adoc [hunk 1]
```diff
@@ -17,16 +17,6 @@ Images for {productname} {productnumber} are available at link:https://www.suse.
...
-.Available Proxy Images
-[cols="3, 3", options="header"]
-|===
-| Architecture | Image Format
-
-| aarch64| qcow2, vmdk, raw
-
-| x86_64  | qcow2, vmdk, raw, Self Installer
-
-|===
```
Comment (admd) [inline modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vm-suma.adoc:20]: We should add a note here that raw images preparation https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation and deployment https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-images-procedure and same for the self install images https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-selfinstal-procedure.

We should do this both for the server and proxy docs.

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 2]
```diff
@@ -0,0 +1,320 @@
+[[install-vm]]
+= Deploy {productname} {productnumber} Proxy as a Virtual Machine
+
+Images for {productname} {productnumber} are available at link:https://www.suse.com/download/suse-manager/[{productname} {productnumber} VM images].
```
Comment (admd) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:4]: We should add an important note here that this preferred way of installation. 

Both for the server and proxy.

**Review by admd — COMMENTED** [review]
Thank you. I have some small suggestions left.

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+[path]``/var/lib/containers/storage/volumes/var-cache`` (Squid)
+Minimum 100 GB
+====
+
+
+[[minimmal.kvm.settings]]
+== Initial KVM Setup
+
+.Procedure: Initial Set Up
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:67]:
```suggestion
.Procedure: Creating Initial Setup
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+. Click btn:[Generate] to register new proxy FQDN in {productname} Server and generate configuration archive with details for container host.
+. After a few moments you are presented with file to download. Save this file locally.
+
+image::suma_proxy_containerized_webui.png[scaledwidth=80%]
+
+[[proxy-setup-containers-transfer-config]]
+== Transfer the Proxy Configuration
+
+The {webui} generates a configuration archive.
+This archive needs to be made available on the Proxy container host.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:211]:
```suggestion
This archive needs to be made available on the proxy container host.
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+
+
+
+[[proxy-setup-containers-transfer-start]]
+== Start the {productname} {productnumber} Proxy
+
+Container can now be started with the [literal]`mgrpxy` command:
+
+[[proc-setup-containers-setup-start]]
+.Task: Start and Check Proxy Status
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:242]:
```suggestion
.Procedure: Start and Check Proxy Status
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+
+To install them run:
+
+----
+zypper in helm mgrpxy
+---- 
+////
+
+
+=== Using a custom container image for a service
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc:301]:
```suggestion
=== Using a Custom Container Image for a Service
```

**Review by keichwa — CHANGES_REQUESTED** [review]

**Pull request merged**